L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -83,6 +83,7 @@
                                kROverZMin=cms.double(0.076),
                                kROverZMax=cms.double(0.58),
                                threshold_histo_multicluster=cms.double(10.),
+                               area_per_triggercell = cms.double(4.9115082E-05), # Hex_Wafer_Area (x/z units) /N_TC (per wafer) = (0.866*((hexWafer_minimal_diameter)*(1./319.))^2 / 48)  
```

Comment:
To be removed from configurable parameters. The comment on the calculation details can be copied in `HGCalHistoSeedingImpl.h `